app/open_json.py

```diff
@@ -0,0 +1,23 @@
+import json
+
+
+with open("config.json") as file:
```

Comment:
```suggestion
with open("app/config.json") as file:
```

Reply:
Why do you need to specify the full path with pytest, but with normal compilation without app/?

Reply:
it's for the tests